cli/src/cast.rs

```diff
@@ -342,10 +342,17 @@ async fn main() -> eyre::Result<()> {
         }
         Subcommands::Run(cmd) => cmd.run().await?,
         Subcommands::SendTx(cmd) => cmd.run().await?,
-        Subcommands::Tx { tx_hash, field, json, rpc } => {
+        Subcommands::Tx { tx_hash, field, raw, json, rpc } => {
             let config = Config::from(&rpc);
             let provider = utils::get_provider(&config)?;
-            println!("{}", Cast::new(&provider).transaction(tx_hash, field, json).await?)
+
+            let raw = raw || matches!(field, Some(TransactionFields::Raw));
+
+            let field = field.and_then(|field| match field {
```

Comment:
Any suggestions on a more eloquent way to convert the enum to an `Option<String>`?

Reply:
hmm, I think the latest changes just added more complexity than we actually need.

I'd be ok to keep the previous solution and just add `--raw` option to the match and document this

Reply:
Simplified it. Any suggestion on where additional documentation would be needed?